toolchain/lower/lower.cpp

```diff
@@ -0,0 +1,19 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/lower/lower.h"
+
+namespace Carbon {
+
+auto LowerResult::Print(llvm::raw_ostream& output) const -> void {
+  output << "TODO: Print IR";
```

Comment:
If you want, this part is easy to wire up:

```suggestion
  module->print(output, nullptr, /*ShouldPreserveUseListOrder=*/true);
```

It might be nice to have two APIs, one used by `dump llvm-ir` and one used by `CARBON_VLOG`. For `VLOG` it should maybe use `module->print(output, nullptr, /*ShouldPreserveUseListOrder=*/false, /*IsForDebug=*/true)` instead of the above.

Reply:
I've switched the call, but I'm not sure what you mean by "two APIs". You've asked this to return llvm::Module directly; we don't control Module's API. Can you please clarify what you're asking for?